### PR TITLE
save interesting unresolved failures

### PR DESCRIPTION
**Pull request by ewlu**
the unresolved failures will be put into a different comment due to character limitations

### CODE REVIEW

File: scripts/aggregate.py [hunk 1]
```diff
@@ -266,17 +279,22 @@ def main():
...
+        all_unresolved[file] = unresolved
         all_new[file] = new
 
     print([i.keys() for i in all_new.values()])
     summary_markdown = failures_to_markdown(failures, args.current_hash, args.patch_name, args.title_prefix)
     resolved_markdown = additional_failures_to_markdown("Resolved",
                                                         all_resolved,
                                                         len(failures['Unresolved']))
+    unresolved_markdown = additional_failures_to_markdown("Unresolved",
+                                                        all_resolved,
```
Comment (patrick-rivos) [inline scripts/aggregate.py:296]:
```suggestion
                                                        all_unresolved,
```

**Review by patrick-rivos — CHANGES_REQUESTED** [review]
Can we save unresolved-failures it to a different file so we don't have to pull it out of the file in the workflow?

**Comment by ewlu** [comment]
>Can we save unresolved-failures it to a different file so we don't have to pull it out of the file in the workflow?

It is pulled out into a different file and isn't included in the current file https://github.com/patrick-rivos/riscv-gnu-toolchain/pull/532/files#diff-0c71e9f2cf7aebf069b30530994236d2fafa34185908f139a62e01a92e29735eR301-R308

File: scripts/aggregate.py [hunk 2]
```diff
@@ -293,7 +293,7 @@ def main():
                                                         all_resolved,
                                                         len(failures['Unresolved']))
     unresolved_markdown = additional_failures_to_markdown("Unresolved",
```
Comment (patrick-rivos) [inline scripts/aggregate.py:295]: Can you move this down below where we write the markdown so it's clear that it isn't included in the args.output_markdown file?

**Review by patrick-rivos — APPROVED** [review]
LGTM after moving the code

**Pull request merged**